Makefile

```diff
@@ -39,10 +43,45 @@ lint: dependencies-frontend
 	$(YARN) lint
 
 assets: build dependencies-frontend
-	$(BINDATA) -pkg asset -o ./server/asset/asset.go `find ./build -type d`
+	chmod -R go=r $(ASSETS_PATH); \
+	$(BINDATA) \
+		-modtime 1 \
+		-pkg asset \
+		-o ./server/asset/asset.go \
+		-prefix $(BASE_PATH) \
+		$(ASSETS_PATH)/...
 
 build: dependencies-frontend
-	REACT_APP_SERVER_URL=$(SERVER_URL) $(YARN) build
+	GENERATE_SOURCEMAP=false REACT_APP_SERVER_URL=$(SERVER_URL) $(YARN) build
+
+validate-commit: assets
```

Comment:
Since this is a common functionality (as already used in [bblfsh/sdk/Makefile#L64](https://github.com/bblfsh/sdk/blob/11595c040c74d033a52d1092d5b9c99f5909c277/Makefile#L64)) it should be moved to [src-d/ci/Makefile.main](https://github.com/src-d/ci/blob/master/Makefile.main), so other projects could be benefited of it

Reply:
they are different. Sdk doesn't use compression (most probably they have a reason for it). But we have to use it. Otherwise, CI fails with out of memory error during build step.

Reply:
mmm... but `::validate-commit` target body is the same in both cases, isn't it?

Reply:
Oh. My bad I thought you are talking about bindata & validate commit (because the latest doesn't make much sense without first). In my opinion, it's better to solve project's needs first and then suggest something company-wide. Like how many src-d projects use bindata? Does it need to be in common repository then? I don't have answers right now. But we need validation in this project.

Reply:
That's because I find better to analyze the problem before starting duplicating already existent code. As far as I can see there are at least two projects needing the same `validate-bindata`, so it make sense to implement it as a CI feature in our CI repo